### PR TITLE
Add workflow alias

### PR DESCRIPTION
**Pull request by pehota**
Add the ability to use aliases for the workflows.

image
 

image

### CODE REVIEW

**Comment by pehota** [comment]
@kanekotic ,
I can't really request a review for some reason. 🤷🏻

**Comment by pehota** [comment]
Seems our prettier configs are different. I will fix that in a separate PR.

**Comment by kanekotic** [comment]
LGTM

File: src/index.ts [hunk 1]
```diff
@@ -1,48 +1,55 @@
...
+import { ObserverManager } from "./main/observers/ObserverManager";
 import "./store";
-import { NotificationManager } from './main/NotificationManager';
-import { dirname, resolve, basename } from 'path';
+import { NotificationManager } from "./main/NotificationManager";
+import { dirname, resolve, basename } from "path";
+import updateElectronApp from "update-electron-app";
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import hasSquirrelStartupEvents from "electron-squirrel-startup";
```
Comment (kanekotic) [inline src/index.ts:12]: @pehota where is this on the package.json

Comment (pehota) [reply src/index.ts:12]: @kanekotic ,
Those had already been there. I only reformatted the import style. I believe those come with electron.

**Comment by kanekotic** [comment]
merged

**Comment by kanekotic** [comment]
@all-contributors please add @pehota for code

**Pull request merged**